lib/season_stat.rb

```diff
@@ -88,7 +88,11 @@ def total_team_wins_by_game_type(team_id, game_type, season)
   def team_win_percentage(team_id, game_type, season)
     total_wins = total_team_wins_by_game_type(team_id, game_type, season).to_f
     total_games = total_team_games_by_game_type(team_id, game_type, season)
-    ((total_wins / total_games) * 100).round(2)
+    if total_games == 0
+      return 0.00
+    else
+      ((total_wins / total_games) * 100).round(2)
+    end
```

Comment:
I could probably refactor this down or separate into a query method later, but this logic is necessary to for biggest_bust to work, otherwise i get Not a Number errors i.e. divide by 0

Reply:
We could probably refactor into a simpler guard, but I think this is perfect for what you need it to do. If we have time we can always come back and revisit